scripts/build-kernel

```diff
@@ -4,31 +4,20 @@
 set -e; eval "$(environment)"
 
 external_dir="${EXTERNAL_DIR?}"
-manifest_dir="${MANIFEST_DIR?}"
 kernel_name="${KERNEL_NAME?}"
 kernel_dir="${external_dir}/kernel/${kernel_name}"
 cores=$(nproc)
 
-mkdir -p "${kernel_dir}"
-cd "${kernel_dir}"
-cat /etc/gitconfig > "$HOME/.gitconfig"
-rm -f "${kernel_dir}/.repo"
-repo init \
-	--manifest-url "${manifest_dir}" \
-	--manifest-name "kernel-${kernel_name}.xml"
-repo sync -c --no-tags --no-clone-bundle --jobs "${cores}"
-repo forall -c 'git reset --hard ; git clean -fdx'
-
 # This seems like it -should- work but the AOSP kernel build system
 # seems to ignore it and generate new keys anyway
 # It is better for determinism and security to disable loadable module support
 # anyway, which some patchsets choose to do.
 #ln -vs \
 #	"${key_dir}/kernel.pem" \
 #	"${kernel_dir}/private/msm-google/certs/signing_key.pem"
-
+cd "${kernel_dir}"
 cat <<-EOF | bash
 	export OUT_DIR="${kernel_dir}/out"
 	export DIST_DIR="${kernel_dir}/dist"
-	bash "${kernel_dir}/build/build.sh" -j$(cores)
+	bash "${kernel_dir}/build/build.sh" -j "$cores"
```

Comment:
`"${cores}"`

Reply:
ha. good catch. no wonder that was not working